flatland/06-chapter.poly.pm

```diff
@@ -19,11 +19,11 @@ Suppose I see two individuals approaching whose rank I wish to ascertain. They a
 
 It will be obvious, to every child in Spaceland who has touched the threshold of Geometrical Studies, that, if I can bring my eye so that its glance may bisect an angle ◊∆{(A)} of the approaching stranger, my view will lie as it were evenly between his two sides that are next to me (viz. ◊∆{CA} and ◊∆{AB}), so that I shall contemplate the two impartially, and both will appear of the same size.
 
-Now in the case of (1) the Merchant, what shall I see? I shall see a straight line ◊∆{D' A' E'}, in which the middle point ◊∆{(A)} will be very bright because it is nearest to me; but on either side the line will shade away ◊emph{rapidly into dimness}, because the sides ◊∆{A' C'} and ◊∆{A' B'} ◊emph{recede rapidly into the fog;} and what appear to me as the Merchant's extremities, viz. ◊∆{D} and ◊∆{E}, will be ◊emph{very dim indeed}.
+Now in the case of (1) the Merchant, what shall I see? I shall see a straight line ◊∆{D A E}, in which the middle point ◊∆{(A)} will be very bright because it is nearest to me; but on either side the line will shade away ◊emph{rapidly into dimness}, because the sides ◊∆{A C} and ◊∆{A B} ◊emph{recede rapidly into the fog;} and what appear to me as the Merchant's extremities, viz. ◊∆{D} and ◊∆{E}, will be ◊emph{very dim indeed}.
 
 ◊margin-figure["img/ch06-fig02.png"]{The Physician}
 
-On the other hand in the case of (2) the Physician, though I shall here also see a line ◊∆{(D' A' E')} with a bright centre ◊∆{(A')}, yet it will shade away ◊emph{less rapidly} into dimness, because the sides ◊∆{(A' C', A' B')} ◊emph{recede less rapidly into the fog;} and what appear to me the Physician's extremities, viz. ◊∆{D'} and ◊∆{E'}, will not be ◊emph{not so dim} as the extremities of the Merchant.
+On the other hand in the case of (2) the Physician, though I shall here also see a line ◊∆{(D' A' E')} with a bright centre ◊∆{(A')}, yet it will shade away ◊emph{less rapidly} into dimness, because the sides ◊∆{(A' C', A' B')} ◊emph{recede less rapidly into the fog;} and what appear to me the Physician's extremities, viz. ◊∆{D'} and ◊∆{E'}, will be ◊emph{not so dim} as the extremities of the Merchant.
```

Comment:
This is a very good catch. It’s true that it looks wrong, but on further investigation it is also an error that persists in nearly every online copy of this text that I have found. I finally found a scan of the original that validates this correction. 
<img width="1403" alt="image" src="https://github.com/otherjoel/try-pollen/assets/1553115/a09861f2-3f8a-4b5b-8fea-aa7c854824d1">